include/triton/Analysis/Utility.h

```diff
@@ -16,6 +16,14 @@ bool maybeAliasOp(Operation *op);
 
 std::string getValueOperandName(Value value, AsmState &state);
 
+template <typename T_OUT, typename T_IN>
+inline SmallVector<T_OUT> convertType(const ArrayRef<T_IN> &in) {
```

Comment:
ArrayRef<T_IN> should be enough since it is already a reference-style class.

Reply:
Thanks for the advice, I will update this function 😃

Reply:
Done